backend/cfg/cfg_with_layout.ml

```diff
@@ -64,6 +64,16 @@ let remove_block t label =
   t.layout <- List.filter (fun l -> not (Label.equal l label)) t.layout;
   t.new_labels <- Label.Set.remove label t.new_labels
 
+let remove_blocks t labels_to_remove =
+  if not (Label.Set.is_empty labels_to_remove)
+  then (
+    Label.Set.iter
+      (fun label -> Cfg.remove_block_exn t.cfg label)
+      labels_to_remove;
```

Comment:
I don't expect huge benefits, but
it may make sense to also have a
bulk removal operation on `Cfg.t`
values, _e.g._ using `filter_map_inplace`.